### PR TITLE
updated file name on download

### PR DESCRIPTION
**Pull request by abilpraju-aot**
# Issue Tracking
https://aottech.atlassian.net/browse/FWF-1865
JIRA: 
Issue Type: BUG/ FEATURE

# Changes
changed the filename when downloading forms/resources also updated the toast message with respect to forms/resources
image
image

### CODE REVIEW

**Review by Bonymol-aot — APPROVED** [review]
LGTM

File: forms-flow-web/src/components/Form/List.js [hunk 1]
```diff
@@ -166,11 +166,18 @@ const List = React.memo((props) => {
...
+  const checkResource = ()=>{
+     return  formCheckList.length === 1 ? t("Resource") : t("Resources");
+    };
+    
   const downloadForms = () => {
-    FileService.downloadFile({ forms: formCheckList }, () => {
+    FileService.downloadFile({ forms: formCheckList}, formType , () => {
       toast.success(
-        `${formCheckList.length} ${formCheckList.length === 1 ? t("Form") : t("Forms")
+        `${formCheckList.length} ${(formType === "form") ? checkForm() : checkResource()
```
Comment (shuhaib-aot) [inline forms-flow-web/src/components/Form/List.js:180]: ` ${ formCheckList.length ==1? t(formType) : t(`${ formType}s`)}`

**Review by shuhaib-aot — CHANGES_REQUESTED** [review]

File: forms-flow-web/src/components/Form/List.js (hunk 1 above)
Comment (shuhaib-aot) [reply forms-flow-web/src/components/Form/List.js:180]: no need to write more functions here it will cause to call stack limit to exceed like RangeError

Comment (abhilash-aot) [reply forms-flow-web/src/components/Form/List.js:180]: Do we really need this? There can be situation of selecting resources and forms together to migrate?

Comment (abilpraju-aot) [reply forms-flow-web/src/components/Form/List.js]: we are selecting the forms and resources seperatly .Thinks needed

Comment (abhilash-aot) [reply forms-flow-web/src/components/Form/List.js]: Is the code adds it seperately or reset the list of selected forms on switching?

Comment (abhilash-aot) [reply forms-flow-web/src/components/Form/List.js]: Here the form download data only deferens at type

**Pull request closed without merging**